src/main/kotlin/com/team4099/robot2025/auto/mode/CenterL4Barge.kt

```diff
@@ -0,0 +1,171 @@
+package com.team4099.robot2025.auto.mode
+
+import choreo.Choreo
+import choreo.trajectory.SwerveSample
+import com.team4099.robot2025.commands.drivetrain.FollowChoreoPath
+import com.team4099.robot2025.commands.drivetrain.ReefAlignCommand
+import com.team4099.robot2025.commands.drivetrain.TargetTagCommand
+import com.team4099.robot2025.config.ControlBoard
+import com.team4099.robot2025.config.constants.Constants
+import com.team4099.robot2025.config.constants.RollersConstants
+import com.team4099.robot2025.subsystems.drivetrain.drive.Drivetrain
+import com.team4099.robot2025.subsystems.elevator.Elevator
+import com.team4099.robot2025.subsystems.superstructure.Superstructure
+import com.team4099.robot2025.subsystems.vision.Vision
+import com.team4099.robot2025.util.AllianceFlipUtil
+import com.team4099.robot2025.util.driver.Jessika
+import edu.wpi.first.wpilibj2.command.Commands.runOnce
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup
+import edu.wpi.first.wpilibj2.command.WaitCommand
+import org.team4099.lib.geometry.Pose2d
+import org.team4099.lib.smoothDeadband
+import org.team4099.lib.units.base.inSeconds
+import org.team4099.lib.units.base.inches
+
+class CenterL4Barge(
+  val drivetrain: Drivetrain,
+  val elevator: Elevator,
+  val superstructure: Superstructure,
+  val vision: Vision
+) : SequentialCommandGroup() {
+  init {
+    addRequirements(drivetrain)
+
+    addCommands(
+      FollowChoreoPath(drivetrain, firstTrajectory),
+      ParallelCommandGroup(
+        superstructure.prepScoreCoralCommand(Constants.Universal.CoralLevel.L4),
+        WaitCommand(1.5)
+          .andThen(
+            ReefAlignCommand(
+              driver = Jessika(),
+              {
+                ControlBoard.forward.smoothDeadband(
+                  Constants.Joysticks.THROTTLE_DEADBAND
+                )
+              },
+              {
+                ControlBoard.strafe.smoothDeadband(
+                  Constants.Joysticks.THROTTLE_DEADBAND
+                )
+              },
+              { ControlBoard.turn.smoothDeadband(Constants.Joysticks.TURN_DEADBAND) },
+              { ControlBoard.slowMode },
+              drivetrain,
+              elevator,
+              superstructure,
+              vision,
+              1
```

Comment:
should add comments for this (which branch just use number or left/right), i think this was very ambiguous in season. small nitpick